### ISSUE TITLE
Only upload questionnaire via js in editor, when there are differences

### ISSUE THREAD

**Issue by TheRadialActive** [sdaps/sdaps_web#19]
Now always the questionnaire edited will be send via POST to the server and writes into the database. Let's reduce those requests via comparing the saved questionnaire with the one in the browser before sending it off.

**Comment by benzea** [comment]
Hmm, is this to prevent the document from being re-rendered subsequently even though nothing actually changed?

**Comment by TheRadialActive** [comment]
That's one thing, but another is, that we now (2 minutes ago) got a new field called "updated_at", which always sets the date, when the model is saved. So if we always write to the database via the editor, we, even when not changing a thing, get a new date at "updated_at", which might be confusing to people at the survey_list page.

**Comment by benzea** [comment]
Yeah, mostly wondered on whether this should happen on the server or client. Doing it on the server seems easier/safer in general, but I also see advantages of doing it client side. And knowing whether to expect a new document in the client is useful.
